sync_events.py

```diff
@@ -0,0 +1,222 @@
+import os
+import pytz
+import glob
+from dataclasses import dataclass
+from pathlib import Path
+
+import requests
+import yaml
+from slugify import slugify
+from dateutil.parser import parse as parse_date
+
+
+@dataclass
+class Event:
+    filename: str
+    pretix_slug: str
+    is_deleted: bool
+    title: str
+    event_date: str
+    event_time: str
+    event_registration: str
+    correlaidx: bool
+    languages: list[str]
+    tags: list[str]
+    description: str
+
+    @classmethod
+    def create(cls, api_event):
+        if api_event.get("slug") is not None:
+            # event
+            pretix_slug = api_event["slug"]
+        else:
+            # subevent
+            pretix_slug = f"{api_event['event']}/{api_event['id']}"
+
+        event = cls(
+            filename=slugify(cls._parse_title(api_event), max_length=50),
```

Comment:
https://github.com/CorrelAid/hugo-website/issues/196#issuecomment-897435958

> also to extract a good slug suggestion from the title would be a challenge.

not sure if this is a "good" slug, but it was my first idea and isn't totally awful.

Reply:
we discussed it and agreed this is ok for now, and that events are stable against title rewording in pretix (we match events based on pretix slug)